src/Nav.php

```diff
@@ -320,31 +453,41 @@ private function renderItemsDropdown(Dropdown $items): Li
      *
      * @param NavLink $item The link to render.
      *
-     * @return A The rendered link.
+     * @return A|Button The rendered link or button.
      */
-    private function renderLink(NavLink $item): A
+    private function renderLink(NavLink $item): A|Button
```

Comment:
Maybe use a name like `createLink`? Cause "render" is usually about getting HTML.

Reply:
done.